Test failure in CI build 2685

The following test appears to have failed:

[#2685](https://circleci.com/gh/cockroachdb/cockroach/2685):

```
I0521 18:44:10.972855     411 multiraft.go:770] group 1: new proposal 13e05119627350896028a72c20e15ca0
I0521 18:44:10.973196     411 multiraft.go:779] node 100000001: group 1 raft ready
I0521 18:44:10.973307     411 multiraft.go:784] HardState updated: {Term:7 Vote:4294967297 Commit:126 XXX_unrecognized:[]}
I0521 18:44:10.974274     411 multiraft.go:787] New Entry[0]: 7/126 EntryNormal 13e05119627350896028a72c20e15ca0: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233850972653386 logical:0 > cmd_id:<wall_time:1432233850971574409 ran
I0521 18:44:10.975199     411 multiraft.go:790] Committed Entry[0]: 7/126 EntryNormal 13e05119627350896028a72c20e15ca0: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233850972653386 logical:0 > cmd_id:<wall_time:1432233850971574409 ran
--- FAIL: TestNodeStatus (1.21s)
	node_test.go:307: 2: AvailableRangeCount does not match expected
		expected: desc:<node_id:"\001" address:<network:"tcp" address:"127.0.0.1:40611" > attrs:<> > store_ids:1 store_ids:2 store_ids:3 range_count:2 started_at:1432233849853911386 updated_at:1432233850526389207 stats:<live_bytes:1 key_bytes:1 val_bytes:1 intent_bytes:0 live_count:14 key_count:14 val_count:40 intent_count:0 intent_age:0 gc_bytes_age:0 sys_bytes:0 sys_count:0 last_update_nanos:0 > leader_range_count:2 replicated_range_count:0 available_range_count:2 
		actual: desc:<node_id:"\001" address:<network:"tcp" address:"127.0.0.1:40611" > attrs:<> > store_ids:2 store_ids:3 store_ids:1 range_count:2 started_at:1432233849853911386 updated_at:1432233850828523233 stats:<live_bytes:1467 key_bytes:1117 val_bytes:8072 intent_bytes:0 live_count:16 key_count:16 val_count:76 intent_count:0 intent_age:0 gc_bytes_age:149 sys_bytes:29618 sys_count:215 last_update_nanos:2864467701320024792 > leader_range_count:2 replicated_range_count:0 available_range_count:1 
=== RUN TestPermEmptyKey
I0521 18:44:10.979380     411 rocksdb.go:88] opening in-memory rocksdb instance
I0521 18:44:10.982332     411 multiraft.go:449] node 100000001 starting
I0521 18:44:10.982744     411 multiraft.go:665] node 100000001 creating group 1
I0521 18:44:10.983233     411 raft.go:390] raft: 100000001 became follower at term 5
I0521 18:44:10.983513     411 raft.go:207] raft: newRaft 100000001 [peers: [100000001], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0521 18:44:10.983773     411 raft.go:464] raft: 100000001 is starting a new election at term 5
--
I0521 18:44:23.296244     411 multiraft.go:779] node 100000001: group 1 raft ready
I0521 18:44:23.296387     411 multiraft.go:784] HardState updated: {Term:6 Vote:4294967297 Commit:19 XXX_unrecognized:[]}
I0521 18:44:23.297108     411 multiraft.go:787] New Entry[0]: 6/19 EntryNormal 13e0511c41053060748d96a5433c8978: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233863295699414 logical:0 > cmd_id:<wall_time:1432233863295610976 rand
I0521 18:44:23.297753     411 multiraft.go:790] Committed Entry[0]: 6/19 EntryNormal 13e0511c41053060748d96a5433c8978: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233863295699414 logical:0 > cmd_id:<wall_time:1432233863295610976 rand
--- PASS: ExampleZoneContentTypes (0.32s)
FAIL
FAIL	github.com/cockroachdb/cockroach/server	16.437s
=== RUN TestStdFlagToPflag
--- PASS: TestStdFlagToPflag (0.00s)
=== RUN TestNoLinkTesting
--- PASS: TestNoLinkTesting (1.38s)
=== RUN: ExampleBasic
I0521 18:44:29.117874     425 rocksdb.go:88] opening in-memory rocksdb instance
I0521 18:44:29.121886     425 multiraft.go:449] node 100000001 starting
I0521 18:44:29.122531     425 multiraft.go:665] node 100000001 creating group 1
I0521 18:44:29.123034     425 raft.go:390] raft: 100000001 became follower at term 5
I0521 18:44:29.123327     425 raft.go:207] raft: newRaft 100000001 [peers: [100000001], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0521 18:44:10.972855     411 multiraft.go:770] group 1: new proposal 13e05119627350896028a72c20e15ca0
I0521 18:44:10.973196     411 multiraft.go:779] node 100000001: group 1 raft ready
I0521 18:44:10.973307     411 multiraft.go:784] HardState updated: {Term:7 Vote:4294967297 Commit:126 XXX_unrecognized:[]}
I0521 18:44:10.974274     411 multiraft.go:787] New Entry[0]: 7/126 EntryNormal 13e05119627350896028a72c20e15ca0: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233850972653386 logical:0 > cmd_id:<wall_time:1432233850971574409 ran
I0521 18:44:10.975199     411 multiraft.go:790] Committed Entry[0]: 7/126 EntryNormal 13e05119627350896028a72c20e15ca0: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233850972653386 logical:0 > cmd_id:<wall_time:1432233850971574409 ran
--- FAIL: TestNodeStatus (1.21s)
	node_test.go:307: 2: AvailableRangeCount does not match expected
		expected: desc:<node_id:"\001" address:<network:"tcp" address:"127.0.0.1:40611" > attrs:<> > store_ids:1 store_ids:2 store_ids:3 range_count:2 started_at:1432233849853911386 updated_at:1432233850526389207 stats:<live_bytes:1 key_bytes:1 val_bytes:1 intent_bytes:0 live_count:14 key_count:14 val_count:40 intent_count:0 intent_age:0 gc_bytes_age:0 sys_bytes:0 sys_count:0 last_update_nanos:0 > leader_range_count:2 replicated_range_count:0 available_range_count:2 
		actual: desc:<node_id:"\001" address:<network:"tcp" address:"127.0.0.1:40611" > attrs:<> > store_ids:2 store_ids:3 store_ids:1 range_count:2 started_at:1432233849853911386 updated_at:1432233850828523233 stats:<live_bytes:1467 key_bytes:1117 val_bytes:8072 intent_bytes:0 live_count:16 key_count:16 val_count:76 intent_count:0 intent_age:0 gc_bytes_age:149 sys_bytes:29618 sys_count:215 last_update_nanos:2864467701320024792 > leader_range_count:2 replicated_range_count:0 available_range_count:1 
=== RUN TestPermEmptyKey
I0521 18:44:10.979380     411 rocksdb.go:88] opening in-memory rocksdb instance
I0521 18:44:10.982332     411 multiraft.go:449] node 100000001 starting
I0521 18:44:10.982744     411 multiraft.go:665] node 100000001 creating group 1
I0521 18:44:10.983233     411 raft.go:390] raft: 100000001 became follower at term 5
I0521 18:44:10.983513     411 raft.go:207] raft: newRaft 100000001 [peers: [100000001], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0521 18:44:10.983773     411 raft.go:464] raft: 100000001 is starting a new election at term 5
--
I0521 18:44:23.296244     411 multiraft.go:779] node 100000001: group 1 raft ready
I0521 18:44:23.296387     411 multiraft.go:784] HardState updated: {Term:6 Vote:4294967297 Commit:19 XXX_unrecognized:[]}
I0521 18:44:23.297108     411 multiraft.go:787] New Entry[0]: 6/19 EntryNormal 13e0511c41053060748d96a5433c8978: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233863295699414 logical:0 > cmd_id:<wall_time:1432233863295610976 rand
I0521 18:44:23.297753     411 multiraft.go:790] Committed Entry[0]: 6/19 EntryNormal 13e0511c41053060748d96a5433c8978: raft_id:1 origin_node_id:4294967297 cmd:<put:<header:<timestamp:<wall_time:1432233863295699414 logical:0 > cmd_id:<wall_time:1432233863295610976 rand
--- PASS: ExampleZoneContentTypes (0.32s)
FAIL
FAIL	github.com/cockroachdb/cockroach/server	16.437s
=== RUN TestStdFlagToPflag
--- PASS: TestStdFlagToPflag (0.00s)
=== RUN TestNoLinkTesting
--- PASS: TestNoLinkTesting (1.38s)
=== RUN: ExampleBasic
I0521 18:44:29.117874     425 rocksdb.go:88] opening in-memory rocksdb instance
I0521 18:44:29.121886     425 multiraft.go:449] node 100000001 starting
I0521 18:44:29.122531     425 multiraft.go:665] node 100000001 creating group 1
I0521 18:44:29.123034     425 raft.go:390] raft: 100000001 became follower at term 5
I0521 18:44:29.123327     425 raft.go:207] raft: newRaft 100000001 [peers: [100000001], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]

```
Please assign, take a look and update the issue accordingly.

--- a/server/node_test.go
+++ b/server/node_test.go
@@ -265,10 +265,9 @@ func TestCorruptedClusterID(t *testing.T) {
 
 // compareNodeStatus ensures that the actual node status for the passed in
 // node is updated correctly. It checks that the Node Descriptor, StoreIDs,
-// RangeCount, StartedAt, LeaderRangeCount, ReplicatedRangeCount and
-// AvailableRangeCount are exactly correct and that the bytes and counts
-// for Live, Key and Val are at least the expected value.  And that UpdatedAt
-// has increased.
+// RangeCount, StartedAt, ReplicatedRangeCount and are exactly correct and that
+// the bytes and counts for Live, Key and Val are at least the expected value.
+// And that UpdatedAt has increased.
 // The latest actual stats are returned.
 func compareStoreStatus(t *testing.T, node *Node, expectedNodeStatus *proto.NodeStatus, testNumber int) *proto.NodeStatus {
 	nodeStatusKey := engine.NodeStatusKey(int32(node.Descriptor.NodeID))
@@ -297,14 +296,8 @@ func compareStoreStatus(t *testing.T, node *Node, expectedNodeStatus *proto.Node
 	if !reflect.DeepEqual(expectedNodeStatus.Desc, nodeStatus.Desc) {
 		t.Errorf("%v: Description does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedNodeStatus, nodeStatus)
 	}
-	if expectedNodeStatus.LeaderRangeCount != nodeStatus.LeaderRangeCount {
-		t.Errorf("%v: LeaderRangeCount does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedNodeStatus, nodeStatus)
-	}
 	if expectedNodeStatus.ReplicatedRangeCount != nodeStatus.ReplicatedRangeCount {
 		t.Errorf("%v: ReplicatedRangeCount does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedNodeStatus, nodeStatus)
-	}
-	if expectedNodeStatus.AvailableRangeCount != nodeStatus.AvailableRangeCount {
-		t.Errorf("%v: AvailableRangeCount does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedNodeStatus, nodeStatus)
 	}
 
 	// There values must >= to the older value.

--- a/storage/client_status_test.go
+++ b/storage/client_status_test.go
@@ -35,9 +35,9 @@ import (
 
 // compareStoreStatus ensures that the actual store status for the passed in
 // store is updated correctly. It checks that the Desc.StoreID, Desc.Attrs,
-// Desc.Node, Desc.Capacity.Capacity, NodeID, RangeCount, LeaderRangeCount,
-// ReplicatedRangeCount and AvailableRangeCount are exactly correct and that
-// the bytes and counts for Live, Key and Val are at least the expected value.
+// Desc.Node, Desc.Capacity.Capacity, NodeID, RangeCount, ReplicatedRangeCount
+// are exactly correct and that the bytes and counts for Live, Key and Val are
+// at least the expected value.
 // The latest actual stats are returned.
 func compareStoreStatus(t *testing.T, store *storage.Store, expectedStoreStatus *proto.StoreStatus, testNumber int) *proto.StoreStatus {
 	storeStatusKey := engine.StoreStatusKey(int32(store.Ident.StoreID))
@@ -72,14 +72,8 @@ func compareStoreStatus(t *testing.T, store *storage.Store, expectedStoreStatus 
 	if expectedStoreStatus.RangeCount != storeStatus.RangeCount {
 		t.Errorf("%v: actual RangeCount does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedStoreStatus, storeStatus)
 	}
-	if expectedStoreStatus.LeaderRangeCount != storeStatus.LeaderRangeCount {
-		t.Errorf("%v: actual LeaderRangeCount does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedStoreStatus, storeStatus)
-	}
 	if expectedStoreStatus.ReplicatedRangeCount != storeStatus.ReplicatedRangeCount {
 		t.Errorf("%v: actual ReplicatedRangeCount does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedStoreStatus, storeStatus)
-	}
-	if expectedStoreStatus.AvailableRangeCount != storeStatus.AvailableRangeCount {
-		t.Errorf("%v: actual RangeCount does not match expected\nexpected: %+v\nactual: %v\n", testNumber, expectedStoreStatus, storeStatus)
 	}
 
 	// Values should be >= to expected values.

